index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" type='text/css'>
+    </head>
+    <body>
+        <div class="imagecontainer">
+            <img class="img img1" src="left_topside_icon.svg" alt="Image geometry on the left top side">
+            <img class="img img2" src="ic_menu.svg" alt="Menu image on the right top side">
+        </div>
+        <div class="textcontainer">
+            <p class="text text1">Charlie Giorgania</p>
+            <p class="text text2">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </div>
+        <div class="rectangle">
+            <div>Start Collaboration</div>
+        </div>
```

Comment:
Kindly use a `section` tag.

Reply:
## Heads-Up: 
Keep in mind whenever you want to add `image_geomery_2` from the Figma design, put in a `section` tag and be aware that the section has the proper background image (decorations are always background images, not `img` tags).

Reply:
As advised by you, I have done the necessary changes